docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -76,27 +76,39 @@ For a list of supported regions, refer to [Cloud provider regions](../../release
 
 The _fault tolerance_ determines how resilient the cluster is to node and cloud zone failures. YugabyteDB Managed provides the following options for providing replication and redundancy:
 
+- **Region level**. Includes 3 nodes spread across multiple regions with a RF of 3. YugabyteDB can continue to do reads and writes even in case of a cloud region failure. This configuration provides the maximum protection for a regional failure.
+
 - **Availability zone level**. Includes a minimum of 3 nodes spread across multiple availability zones with a [replication factor](../../../architecture/docdb-replication/replication/) (RF) of 3. YugabyteDB can continue to do reads and writes even in case of a cloud availability zone failure. This configuration provides the maximum protection for a data center failure.
 
 - **Node level**. Includes a minimum of 3 nodes deployed in a single availability zone with a RF of 3. YugabyteDB can continue to do reads and writes even in case of a node failure, but this configuration is not resilient to cloud availability zone outages.
 
-- **Region level**. Yugabyte supports multi-region clusters with regional fault tolerance. Contact {{% support-cloud %}} for help configuring regional fault tolerance.
-
 Although you can't change the cluster fault tolerance after the cluster is created, you can scale horizontally as follows:
 
+- For Region level, you can add or remove nodes in increments of 1 per region; all regions have the same number of nodes.
 - For Availability zone level, you can add or remove nodes in increments of 3.
 - For Node level, you can add or remove nodes in increments of 1.
 
-For production clusters, Availability zone level is recommended.
+For production clusters, multi-region clusters providing region level fault tolerance  is recommended.
```

Comment:
Is this true? Aren't there latency considerations for customers depending on their requirements? I've seen that a lot of customers for YBA for example go with AZ level fault tolerance for this reason.

Reply:
Yeah I would replace this with something more like "Which fault tolerance level is appropriate depends heavily on your application architecture and design.  If you are unsure of the correct setting for your situation, please contact Yugabyte Support and we will work with you to understand your use case and provide guidance  on what settings to use."